microovn/api/endpoints.go

```diff
@@ -16,7 +17,9 @@ var Server = rest.Server{
 	Resources: []rest.Resources{
 		{
 			PathPrefix: types.APIVersion,
-			Endpoints: []rest.Endpoint{servicesCmd,
```

Comment:
Making the existing `servicesCmd` function public and renaming the module is a logical unit of change in itself. Please put that into a separate commit so that it is easier to discern between reorganization/refactoring of existing code and new code during review.